src/css/components/_transient-button.scss

```diff
@@ -0,0 +1,62 @@
+.video-js .vjs-transient-button {
+  position: absolute;
+  height: 3em;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: rgba(50, 50, 50, 0.5);
+  cursor: pointer;
+  opacity: 1;
+  transition: opacity 1s;
+}
+
+.video-js:not(.vjs-has-started) .vjs-transient-button {
+  display: none;
+}
+
+.video-js .vjs-transient-button span {
+  padding: 0 0.5em;
+}
+
+.video-js .vjs-transient-button.left {
+  left: 5px;
+}
+
+.video-js .vjs-transient-button.right {
+  right: 5px;
+}
+
+.video-js .vjs-transient-button.top {
+  top: 5px;
+}
+
+.video-js .vjs-transient-button.bottom {
+  bottom: 35px;
+}
+
+.video-js.vjs-layout-large .vjs-transient-button {
+  height: 5em;
+  padding: 0 1em;
+  font-size: 120%;
+}
+
+.video-js.vjs-layout-x-large .vjs-transient-button,
+.video-js.vjs-layout-huge .vjs-transient-button {
+  height: 8em;
+  padding: 0 1.5em;
+
+  font-size: 150%;
+}
```

Comment:
I would have been inclined to do nothing in this case, in order to leave freedom to the developer, and keep the same visual consistency as the control bar.

What was the intention behind it? Maybe I'm missing a bit of context, hence my comment.

Reply:
It's to avoid small buttons on large players. Maybe we could leave this out, I do want to keep the styling functional but relatively light so they can be customised.